components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -202,18 +202,37 @@
                     {{ locale }}
                 </option>
             </select>
+            <h2
+                class="mod-healthcare-professional-section
+                 my-3.5 text-start text-primary-text text-2xl font-bold font-sans leading-normal"
+            >
+                {{ $t("modHealthcareProfessionalSection.facilities") }}
+            </h2>
+            <!-- This is just an example how to use it -->
+            <ModFacilitySearchbar v-model="selectedFacitilies" />
```

Comment:
just realized this is misspelled. 
should be `selectedFacilities` not `facitilies`